nemo/collections/asr/parts/mixins/transcription.py

```diff
@@ -480,6 +481,12 @@ def _transcribe_input_processing(self, audio, trcfg: TranscribeConfig):
 
         # Check if audio is a list of strings (filepaths or manifests)
         if isinstance(audio[0], str):
+            trcfg._internal.manifest_filepath = None
```

Comment:
You need to add this argument to the internal config of transcribe above

Reply:
Done, thank you!